modules/guides/pages/connect.adoc

```diff
@@ -0,0 +1,422 @@
+
+. Pass a `certpath` query parameter to the connection url, after the bucket name.
+
+'''
+
+The example below connects to the `travel-sample` bucket with a client certificate, and performs a `ping` to check what services are running in a single-node _secured_ cluster environment.
+
+[source,shell]
+----
+include::example$connect/connect-cbc.sh[tag=cbc-connect-cert]
+----
+
+{cbc-api-url}
+--
+
+.NET::
++ 
+--
+. Call the `WithX509CertificateFactory()` method on a `ClusterOptions()` object.
+
+. Provide the certificate store information to the `WithX509CertificateFactory()` method.
+
+. Call the `ConnectAsync()` method and pass it the cluster options object.
+
+'''
+
+{connect-tls-example}
+
+It is assumed that a valid client certificate and certificate store have been setup.
+
+[source,csharp]
+----
+include::dotnet-sdk:howtos:example$Auth.cs[tag=auth,indent=0]
+----
+
+{github}
+
+For further details, refer to {dotnet-api-url}/Couchbase.Cluster.html[Cluster^].
+--
+
+Java::
++ 
+--
+. Load a Java keystore file containing your client certificate and pass it to the `CertificateAuthenticator.fromKeyStore()` method along with the required password.
+
+. Call the `Cluster.connect()` method and pass the connection string along with cluster options containing the `CertificateAuthenticator` object previously created.
+
+'''
+
+{connect-tls-example}
+
+It is assumed that a valid client certificate and a Java keystore have been setup.
+
+[source,java]
+----
+include::java-sdk:howtos:example$Auth.java[tag=certauth,indent=0]
+----
+
+{github}
+
+For further details, refer to {java-api-url}/Cluster.html[Cluster^].
+--
+
+Node.js::
++ 
+--
+Call the `connect()` function with a connection url, and a `ConnectOptions` object containing a certificate `trustStorePath` and user credentials.
+
+'''
+
+{connect-tls-example}
+
+It is assumed that a valid client certificate has been setup.
+
+[source,nodejs]
+----
+include::nodejs-sdk:howtos:example$auth.js[tag=tls-cacert,indent=0]
+----
+
+{github}
+
+For further details, refer to {nodejs-api-url}/Cluster.html[Cluster^].
+--
+
+Python::
++ 
+--
+. Call the `Cluster.connect()` function with a connection url, and a `ClusterOptions` object containing a `PasswordAutheticator`.
+
+. Provide a username and password to the `PasswordAutheticator`.
+
+. Supply a certificate path to the `PasswordAutheticator`.
+
+'''
+
+{connect-tls-example}
+
+It is assumed that a valid client certificate has been setup.
+
+[source,python]
+----
+include::python-sdk:howtos:example$managing_connections.py[tag=tls,indent=0]
+----
+
+{github}
+
+For further details, refer to {python-api-url}#cluster-object[Cluser^].
+--
+====
+
+== Connecting to Couchbase Capella
+
+To connect to the database via Couchbase Capella cloud, ensure that you have setup some credentials for your cloud setup, including a client certificate.
```

Comment:
Inline links to the cloud documentation for setting up credentials for cloud setup and client certificate.